hack/testing-olm/test-001-operator-sdk-e2e.sh

```diff
@@ -17,7 +17,7 @@ if [ ! -d $test_artifact_dir ] ; then
 fi
 
 
-TEST_NAMESPACE="${TEST_NAMESPACE:-e2e-test-${RANDOM}}"
+TEST_NAMESPACE="openshift-operators-redhat"
```

Comment:
this should be set in the makefile instead of in each file because of how the cert generation works